tests/Database/DatabaseFunctionalTest.php

```diff
@@ -211,6 +213,29 @@ public function testModifyCollection(): void
         }
     }
 
+    public function testRenameCollection(): void
+    {
+        $renamedCollection = $this->getCollectionName() . '.renamed';
+        $renamedNamespace = $this->getDatabaseName() . '.' . $renamedCollection;
+        $operation = new DropCollection($this->getDatabaseName(), $renamedCollection);
+        $operation->execute($this->getPrimaryServer());
+
+        $bulkWrite = new BulkWrite();
+        $bulkWrite->insert(['_id' => 1, 'x' => 'foo']);
+
+        $writeResult = $this->manager->executeBulkWrite($this->getNamespace(), $bulkWrite);
+        $this->assertEquals(1, $writeResult->getInsertedCount());
+
+        $commandResult = $this->database->renameCollection($this->getNamespace(), $renamedNamespace);
+        $this->assertCommandSucceeded($commandResult);
+        $this->assertCollectionCount($this->getNamespace(), 0);
+        $this->assertCollectionCount($renamedNamespace, 1);
```

Comment:
Oh I used `$this->assertCollectionCount()` because I saw that the `testDrop()` function used it, but I realize now that function is testing dropping a database, not a collection. I'll change `testRenameCollection()` to use the existence assertions.